manifests/repo/redhat.pp

```diff
@@ -10,4 +10,5 @@
   $yum_repo = 'remi_php56',
 ) {
   contain "::yum::repo::${yum_repo}"
+  contain "::yum::repo::remi"
```

Comment:
We've put `puppet-lint` into full pedantic mode so that the code quality metric on the Puppet Forge is always at 100%. :panda_face: 

This is the relevant warning that is treated as an error:
```
manifests/repo/redhat.pp - WARNING: double quoted string containing no variables on line 13
```

Sorry, you'll have to use single quotes here. :)